File: internal/extract/testdata/csharp/basic.cs
using System;
using System.Collections.Generic;

namespace MyApp {
    public class Animal {
        public Animal(string name) {
        }

        public void Speak() {
            Console.WriteLine("hello");
        }
    }

    public class Dog : Animal {
        public Dog(string name) : base(name) {
        }

        public void Speak() {
            Console.WriteLine("woof");
        }
    }

    public interface IGroomable {
        void Groom();
    }

    public struct Point {
        public int X;
        public int Y;
    }

    public enum Color {
        Red, Green, Blue
    }
}

File: internal/extract/testdata/java/basic.java
import java.util.List;

public class Animal {
    private String name;

    public Animal(String name) {
        this.name = name;
    }

    public void speak() {
        System.out.println("hello");
    }
}

class Dog extends Animal implements Groomable {
    public Dog(String name) {
        super(name);
    }

    public void speak() {
        this.wag();
        System.out.println("woof");
    }
}

interface Groomable {
    void groom();
}

enum Color {
    RED, GREEN, BLUE
}
